tests/ios/MASVS-PLATFORM/MASTG-TEST-0070.md

```diff
@@ -405,4 +405,6 @@ $ xcrun swift-demangle S10TelegramUI15openExternalUrl7account7context3url05force
 
 Resulting in:
 
-```default
+```bash
```

Comment:
Please go back to v1.5.0 and get the missing text from the original test in Document/0x06h-Testing-Platform-Interaction.md 

```suggestion
Resulting in:

...

This knowledge should help you when testing apps supporting Handoff.
```